lms/static/js/Markdown.Editor.js

```diff
@@ -1040,107 +1057,144 @@
         // isCancel is false if we are going to keep the text.
         var close = function (isCancel) {
             util.removeEvent(doc.body, "keydown", checkEscape);
-            var text = input.value;
+            var url = urlInput.value.trim();
+            var description = descInput.value.trim();
+
+            // reset the form to show no errors.
+            urlInput.classList.remove('has-error');

```

Comment:
Nit: I'd factor this out into a ``clearFormErrorMessages`` function both to make this function simpler to understand, and to make the code self describing (so then the comment isn't necessary.